src/lunariumcmd.src

```diff
@@ -1629,12 +1634,13 @@ cmds.rshell=function(sin)
     out={}
     out.bool=2
 	
-	if shelltype != "file" then
-		if not comp.is_network_active() then
-			return error("Internet connection required")
-		end if
-	end if
+	//if shelltype != "file" then // And why this check for the commands that already check "origShell"? The origShell check will automatically filter these out and you can do a simple one-line check for internet directly afterwards
```

Comment:
Because honestly, I don't have 100% understanding of what your code does. I have started this project back when I had little understanding of MiniScript and Grey Hack. More over, it actually is hard to maintain code base without(or almost without) any documentation or comment blocks.

Reply:
understandable, i'll add more comments next time